docs/en/papers/stam-cn.md

```diff
@@ -0,0 +1,160 @@
+STAM：针对大型分布式应用系统的拓扑自动检测方法
+
+[![英文版本](https://img.shields.io/badge/Original_Paper-en-green)](stam.md)
```

Comment:
Why not consider hosting the English version here and pointing the Chinese version to an external link?

Reply:
Let me remove them all. I don't have any purpose, just copy/paste the old one.

Reply:
Removed CN version.